tests/integration/serializers/json-api-serializer-test.js

```diff
@@ -165,6 +165,8 @@ testInDebug('Warns when normalizing payload with unknown type included', functio
   }, /Encountered a resource object with type "unknown-types", but no model was found for model name "unknown-type"/);
 });
 
+// TODO This test probably shouldn't pass in the case where company data is unknown type
+/*
```

Comment:
we should `skip` tests instead of commenting them out or we are likely to forget to fix them or address the reasons they were removed.